contracts/CRToken.sol

```diff
@@ -0,0 +1,224 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.6.0) (token/ERC20/ERC20.sol)
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+import "@openzeppelin/contracts/utils/Context.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract CRToken is Context, Ownable, IERC20, IERC20Metadata {
+    mapping(address => uint256) private _balances;
+
+    mapping(address => mapping(address => uint256)) private _allowances;
+
+    uint256 private _totalSupply;
+
+    string private _name;
+    string private _symbol;
+
+    /**
+     * Mint the cr token against the staked asset
+     * It cannot be called by any one other than the smart contract
+     */
+    function mint(address _to,uint256 _amount)external onlyOwner {
+    _mint(_to,_amount);
+    }
+
+    /**
+     * Burn the cr token after withdrawal of the staked asset
+     * It cannot be called by any one other than the smart contract
+     */
+    function burn(address _to,uint256 _amount)external onlyOwner {
+    _burn(_to,_amount);
```

Comment:
Shouldn't be an indentation here: A tab or space ? Or was this done automatically by your code formatter?